src/components/Avatar.jsx

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+
+function getUserInitials(name) {
+  if (!name) return 'GU';
+  return name
+    .split(' ')
+    .map((part) => part[0])
+    .join('')
+    .toUpperCase()
+    .slice(0, 2);
+}
+
+export function Avatar({ data, size, fontSize }) {
+  // Handle different data structures for user and member
+  const name = data?.fullname || data?.username || '';
+  const imageUrl = data?.imgUrl || data?.avatarUrl || '';
+
+  const initials = getUserInitials(name);
+
+  return (
+    <AvatarWrapper
+      $imageUrl={imageUrl}
+      size={size}
+      fontSize={fontSize}
+      title={name || 'User'}
+      role="img"
+      aria-label={name || 'User avatar'}
+    >
+      {imageUrl ? null : initials}
```

Comment:
if you have imageUrl you return null?
better to do an img tag and add the imageUrl there.
not forget if not have image, there is an initial letters and background-color

Reply:
you dont pass the bgColor prop